Meros.nimble

```diff
@@ -25,12 +25,69 @@ requires "stint"
 requires "nimcrypto"
 requires "normalize"
 
+# Test Dependencies
```

Comment:
"# " < "#" in my opinion. If this is a sticking point/majority shared opinion, the discussion becomes updating the whole codebase, but I'd not like to fragment comment usage across the codebase.